src/main/java/com/example/Mini/product/Product.java

```diff
@@ -0,0 +1,25 @@
+package com.example.Mini.product;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.*;
+
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    private String productName;
+    private Long price;
+    private Long quantity;
+    private String imageUrl;
```

Comment:
các từ viết tắt thì phải viết hoa tất hoặc thường tất
```suggestion
    private String imageURL;
```

Reply:
Vâng đã tiếp thu